config/manifests/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ resources:
 - ../default
 - ../samples
 - ../scorecard
+- ../cert-manager-prometheus
```

Comment:
lets add the `../prometheus` here as well.

Reply:
../prometheus is for the operator's monitoring, and we are considering the operand's monitoring in this PR.

Reply:
Yes, the folder can be re-used for both operand and operator servicemonitor. Let the distinction be in the file names as I suggest above.